dbms/src/Flash/Mpp/ExchangeReceiver.h

```diff
@@ -213,6 +214,13 @@ class ExchangeReceiverBase
 
     bool collected = false;
     int thread_count = 0;
+    uint64_t fine_grained_shuffle_stream_count;
+
+    /// True when:
+    ///   1. we are in disaggregated tiflash mode
+    ///   2. and this receiver is receiving data from tiflash_storage node.
+    /// If it's true, when this ExchangeReciever is canceled, need to cancel MPPTasks in tiflash_storage node.
+    bool is_tiflash_storage_receiver;
```

Comment:
Will `is_receiver_for_tiflash_storage` be more explicit? The current naming may be misunderstood as "a receiver on tiflash storage node".

Reply:
Done